msm/msm

```diff
@@ -23,8 +23,6 @@
 # This script assists in the installation and management of
 # skills loaded from Github.  
 
-set -e
```

Comment:
This needs to be remain to make sure download errors makes the script exit with error